ai-service/tests/conftest.py

```
@@ -1,48 +1,50 @@
+        initialize_model()
+        yield
 
 
 @pytest.fixture(autouse=True)
 def setup_test_context_and_clean_db(
-    monkeypatch: pytest.MonkeyPatch,
     request: pytest.FixtureRequest,
-    db_test_collection: chromadb.Collection,
 ) -> Generator[None, None, None]:
     # Create a fake repo URL based on the test module name
     module = getattr(request, "module")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)